TRANSFORMERS/VECTOR_MANIPULATION/REMOVE_DUPLICATES_VECTOR/REMOVE_DUPLICATES_VECTOR.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
The studio people want us to only import what we need (here `from numpy import unique`)

Reply:
So change that for all the nodes